LibraBFT/Impl/Properties/VotesOnceWU.agda

```diff
@@ -0,0 +1,102 @@
+{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
+
+   Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+   Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.oracle.com/licenses/upl
+-}
+{-# OPTIONS --allow-unsolved-metas #-}
+
+-- This module proves the two "VotesOnce" proof obligations for our fake handler
+
+open import Optics.All
+open import LibraBFT.Prelude
+open import LibraBFT.Lemmas
+open import LibraBFT.Base.PKCS
+
+import      LibraBFT.Concrete.Properties.VotesOnce as VO
+
+open import LibraBFT.Impl.Consensus.Types hiding (EpochConfigFor)
+open import LibraBFT.Impl.Util.Crypto
+open import LibraBFT.Impl.Handle.Properties                               sha256 sha256-cr
+open import LibraBFT.Impl.Properties.Aux
+open import LibraBFT.Concrete.System impl-sps-avp
+open import LibraBFT.Concrete.System.Parameters
+open        EpochConfig
+open import LibraBFT.Yasm.Yasm (ℓ+1 0ℓ) EpochConfig epochId authorsN ConcSysParms NodeId-PK-OK
+open        Structural impl-sps-avp
+open import LibraBFT.Impl.Properties.VotesOnce
+
+-- In this module, we (will) prove the two implementation obligations for the VotesOnce rule.  Note
+-- that it is not yet 100% clear that the obligations are the best definitions to use.  See comments
+-- in Concrete.VotesOnce.  We will want to prove these obligations for the fake/simple
+-- implementation (or some variant on it) and streamline the proof before we proceed to tacke more
+-- ambitious properties.
+
+module LibraBFT.Impl.Properties.VotesOnceWU where
```

Comment:
The name is a bit cryptic.  I guess WU is "Without Unwind"?  Maybe replace "WU" with "Direct"?  Please also put a comment in the top of this file mentioning that it duplicates the original proof, and we are keeping both for now to highlight the different approaches.